README.md

```diff
@@ -3,9 +3,9 @@ Takeda Integrated Health Systems
 
 ## Environments
 
--Preview: https://main--takeda-ihs--hlxsites.hlx.page/
+-Preview: https://remove-hardcoded-text-from-hero-card--takeda-ihs--hlxsites.hlx.page/
 
--Live: https://main--takeda-ihs--hlxsites.hlx.live/
+-Live: https://remove-hardcoded-text-from-hero-card--takeda-ihs--hlxsites.hlx.live/
```

Comment:
This shouldn't change on a PR by PR basis. I think you meant to put this in the PR description?

Reply:
Fixed